src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -692,8 +697,9 @@ public void update() {
         }
 
         actions.put(Actions.TOGGLE_PREVIEW, (BaseAction) () -> {
-            boolean enabled = !Globals.prefs.getBoolean(JabRefPreferences.PREVIEW_ENABLED);
-            Globals.prefs.putBoolean(JabRefPreferences.PREVIEW_ENABLED, enabled);
+            PreviewPreferences previewPreferences = new PreviewPreferences(Globals.prefs);
+            boolean enabled = !previewPreferences.isPreviewEnabled();
```

Comment:
The boolean stores if the preview is enabled.
This line simply inverses what is saved in the preferences.